cv/qc-cv.obo

```diff
@@ -18,1967 +18,341 @@ import: https://raw.githubusercontent.com/ISA-tools/stato/dev/releases/1.4/stato
-is_a: MS:4000010 ! ID free
-is_a: MS:4000001 ! QC metric
-is_a: MS:4000004 ! n-tuple
-relationship: has_units UO:0000106 ! hertz
-
-[Term]
-id: MS:4000147
-name: MS1 ion collection time Q1, Q2, Q3
-def: "From the distribution of ion injection times (MS:1000927) for MS1, the quartiles Q1, Q2, Q3" [PSI:QC]
-comment: Injection time distribution can be used to gauge the suitability of used instrument settings for the sample content used.
-is_a: MS:4000004 ! n-tuple
-relationship: has_units STATO:0000167 ! first quartile
-relationship: has_units STATO:0000574 ! center value
-relationship: has_units STATO:0000170 ! third quartile
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_value_concept STATO:0000574 ! center value
+synonym: "MS1-2B" NARROW []
 
 [Term]
-id: MS:4000148
-name: MS1 ion collection time mean
-def: "From the distribution of ion injection times (MS:1000927) for MS1, the mean" [PSI:QC]
-comment: Injection time distribution can be used to gauge the suitability of used instrument settings for the sample content used.
+id: MS:4000095
+name: median S/N for MS1 spectra in the shortest RT range in which half of the spectra are identified
+def: "Median S/N for MS1 spectra whose RT is in the shortest RT range in which a half of all identified MS2 at given FDR threshold are." [PSI:QC]
 is_a: MS:4000003 ! single value
-is_a: MS:4000010 ! ID free
-is_a: MS:4000001 ! QC metric
-relationship: has_units STATO:0000401 ! sample mean
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_value_concept STATO:0000574 ! center value
+relationship: has_units STATO:0000300 ! dimensionless ratio
 
 [Term]
-id: MS:4000149
-name: MS1 ion collection time sigma
-def: "From the distribution of ion injection times (MS:1000927) for MS1, the sigma value" [PSI:QC]
-comment: Injection time distribution can be used to gauge the suitability of used instrument settings for the sample content used.
+id: MS:4000096
+name: median of TIC values in the shortest RT range in which half of the peptides are identified
+def: "Median of TIC values whose RT is in the shortest RT range in which a half of all identified MS2 at given FDR threshold are." [PSI:QC]
 is_a: MS:4000003 ! single value
-is_a: MS:4000010 ! ID free
-is_a: MS:4000001 ! QC metric
-relationship: has_units STATO:0000237 ! standard deviation
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_value_concept STATO:0000574 ! center value
 
 [Term]
-id: MS:4000150
-name: MS1 ion collection time low outliers (<Q1-1.5*IQR)
-def: "From the distribution of ion injection times (MS:1000927) for MS1, the list of outliers below Q1-1.5*IQR" [PSI:QC]
-comment: Injection time distribution can be used to gauge the suitability of used instrument settings for the sample content used.
-is_a: MS:4000001 ! QC metric
-is_a: MS:4000010 ! ID free
+id: MS:4000097
+name: charged identifications ratio
```

Comment:
"ratio of 1+/3+/4+ to 2+ charged identifications"
Are only these 3 recorded here, or "ratio of charged identifications to 2+ charged"?